Plugins/Mute/Plugin.cs

```diff
@@ -0,0 +1,55 @@
+﻿using SharedLibraryCore;
+using SharedLibraryCore.Interfaces;
+
+namespace Mute;
+
+public class Plugin : IPlugin
+{
+    public Plugin(IMetaServiceV2 metaService)
+    {
+        DataManager = new DataManager(metaService);
+    }
+
+    public string Name => "Mute";
+    public float Version => (float) Utilities.GetVersionAsDouble();
+    public string Author => "Amos";
+
+    public const string MuteKey = "IW4MMute";
+
+    public static DataManager? DataManager;
```

Comment:
If you initialize this in the constructor it will never be null, remove the nullable qualifier